packages/xo-server/src/xapi/index.js

```diff
@@ -2339,5 +2340,17 @@ export default class Xapi extends XapiBase {
     )
   }
 
-  // =================================================================
+  async _checkDateConsistency (hostRef) {
```

Comment:
`_assertConsistentHostServerTime`?

- `assert`: throw if the condition is not validated
- `HostServerTime`: clearer than simply `Date`